translator/translate/otel/exporter/prometheusremotewrite/translator_test.go

```diff
@@ -33,7 +33,7 @@ func TestTranslator(t *testing.T) {
 					"metrics_destinations": map[string]interface{}{},
 				},
 			},
-			wantErr: &common.MissingKeyError{ID: tt.ID(), JsonKey: AMPSectionKey},
+			wantErr: &common.MissingKeyError{ID: tt.ID(), JsonKey: AMPSectionKey + " or " + common.ConfigKey(AMPSectionKey, common.WorkspaceIDKey)},
```

Comment:
Can you update the test so it has the `amp` section, but not the `workspace_id`?